collectors/python.d.plugin/samba/metadata.yaml

```diff
@@ -3,52 +3,116 @@ meta:
   module_name: samba
   monitored_instance:
     name: Samba
-    link: ''
+    link: https://www.samba.org/samba/
     categories:
-    - data-collection.storage-mount-points-and-filesystems
-    icon_filename: 'samba.jpg'
+      - data-collection.storage-mount-points-and-filesystems
+    icon_filename: "samba.jpg"
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
-    description: ''
-  keywords: []
+    description: ""
+  keywords:
+    - samba
+    - file sharing
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Monitor Samba performance for optimal network file sharing operations. Netdata provides real-time insights and alerts for efficient file sharing.'
-    method_description: ''
+    metrics_description: "This collector monitors the performance metrics of Samba file sharing."
+    method_description: |
+      It is using the `smbstatus` command-line tool.
+
+      Executed commands:
+
+      - `sudo -n smbstatus -P`
   supported_platforms:
     include: []
     exclude: []
-  multi_instance: true
+  multi_instance: false
   additional_permissions:
-    description: ''
+    description: |
+      `smbstatus` is used, which can only be executed by `root`. It uses `sudo` and assumes that it is configured such that the `netdata` user can execute `smbstatus` as root without a password.
```

Comment:
@Ancairon we have `yaml` reporting error with line-length that I understand can be ignored, please, can you confirm?

Reply:
yes, we can ignore the line length error